cifar10-multiannotator/multiannotator_labels.ipynb

```diff
@@ -4728,23 +359,23 @@
    ],
    "source": [
     "consensus_gtruth_accuracy = [model_results[i][\"consensus_gtruth_accuracy\"] \n",
-    "                             for i in range(itter)] + [final_consensus_accuracy]\n",
+    "                             for i in range(rounds)] + [final_consensus_accuracy]\n",
     "\n",
-    "model_gtruth_accuracy = [model_results[i][\"model_gtruth_accuracy\"] for i in range(itter)]\n",
+    "model_gtruth_accuracy = [model_results[i][\"model_gtruth_accuracy\"] for i in range(rounds)]\n",
     "\n",
     "\n",
     "# plot prc\n",
     "plt.rcParams[\"figure.figsize\"] = (17,6)\n",
     "plt.show()\n",
     "\n",
-    "plt.subplot(1, 2, 1)\n",
-    "plt.plot(range(-1, itter),consensus_gtruth_accuracy)\n",
+    "ax1 = plt.subplot(1, 2, 1)\n",
+    "plt.plot(range(0, rounds+1),consensus_gtruth_accuracy)\n",
     "plt.xlabel(\"Rounds\", fontsize=14)\n",
     "plt.ylabel(\"Accuracy of Consensus Labels\", fontsize=14)\n",
     "plt.title(\"Consensus Label vs Ground Truth Labels\", fontsize=14, fontweight=\"bold\")\n",
     "\n",
-    "plt.subplot(1, 2, 2)\n",
-    "plt.plot(range(0, itter), model_gtruth_accuracy)\n",
+    "plt.subplot(1, 2, 2, sharex=ax1, sharey=ax1)\n",
+    "plt.plot(range(0, rounds), model_gtruth_accuracy)\n",
```

Comment:
[nit] I think this might make more sense if it's `plt.plot(range(0, rounds + 1), model_gtruth_accuracy)` because there is no model trained in round 0? 
Might also be helpful to have a quick sentence explaining why it is missing on datapoint when compared to the plot on its left. Something like "the first round of consensus labels were determined purely using annotator labels and we did not train a model that round"